### ISSUE TITLE
Modifying Pytorch Model Classes and DataLoaders to work with Graphcore

### ISSUE THREAD

**Issue by armaneshaghi** [graphcore/tutorials#15]
I am using a package that works on top of PyTorch. Going back and modifying code such that loss computations are part of forward passes encapsulated into the models and run on IPUs or changing data loaders are non-trivial tasks and may need me to re-write a whole Python package for it. 

Can I modify existing classes that I can return from existing functions? If so are there tutorials for it?

A couple of example scripts in Pytorch is here that I hope to be able to run on Graphcore: 

https://github.com/Project-MONAI/tutorials/blob/main/modules/dynunet_pipeline/inference.py

https://github.com/Project-MONAI/tutorials/blob/main/modules/dynunet_pipeline/train.py

**Comment by armaneshaghi** [comment]
Being followed through on the Slack channel.

**Comment by payoto** [comment]
Thanks for keeping track of this @armaneshaghi,
For anyone else wanting to follow up the thread in question is here : https://graphcorecommunity.slack.com/archives/C03EYU38LM9/p1677316737362679

And you can join the Graphcore IPU community here:
https://www.graphcore.ai/join-community
